src/styles/blocks/page.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  font-family: 'Space Grotesk', sans-serif;
+  font-weight: 700;
+  color: $main-text-color;
+  scroll-behavior: smooth;
```

Comment:
scroll-behavior should be applied on the 'html' or 'body' element to work properly.